TCAT/Controllers/HomeViewController.swift

```diff
@@ -507,7 +528,48 @@ extension HomeViewController: AddFavoritesDelegate {
     }
 }
 
+extension HomeViewController: WhatsNewDelegate {
+    func okButtonPressed() {
+        userDefaults.set(true, forKey: Constants.UserDefaults.whatsNewDismissed)
+        let lastRowIndex = IndexPath(row: 0, section: sections.count - 1)
```

Comment:
are we sure sections.count will never be 0?

Reply:
Yeeeaaaa so I tested that a few days ago and it crashed so I’m working to figure out how else to do it